File: cs50_examples/week_1/mario.c
#include<stdio.h>
#include <cs50.h>

int main()
{
	for (int raw = 0; raw < 3; raw++)
	{
		for (int col = 0; col < 3; col++)
		{
			printf("#");
		}
		printf("\n");
	}
}

File: cs50_psets/week_6/sentimental-mario-more/mario.py
while True:
    try:
        height = int(input("Height: "))
        if height > 0 and height <= 8:
            break
    except ValueError:
        continue


for raw in range(height):
    space = height  
    space = space - (raw + 1)
    print(" " * space, end="")

    for col in range(raw + 1):
        print("#", end="")

    print("  ", end="")

    for col in range(raw + 1):
        print("#", end="")
    
    print()

